lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -22,11 +22,15 @@ class DescribeClass < Cop
         MSG = 'The first argument to describe should be '\
               'the class or module being tested.'.freeze
 
-        def_node_matcher :valid_describe?, <<-PATTERN
-          {
-            (send {(const nil? :RSpec) nil?} :describe const ...)
-            (send {(const nil? :RSpec) nil?} :describe)
-          }
+        FIRST_ARGUMENT_REQUIRED_MSG =
+          'The first argument to describe is required.'.freeze
+
+        def_node_matcher :describe_with_const?, <<-PATTERN
+          (send {(const nil? :RSpec) nil?} :describe const ...)
+        PATTERN
+
+        def_node_matcher :first_argument_empty?, <<-PATTERN
```

Comment:
is it really necessary to use a node_matcher for this? Can you just check `node.arguments?` instead

Reply:
Good point. I changed it.